examples/examples.go

```diff
@@ -40,12 +42,49 @@ func ReadFile(path string) []byte {
 	return contents
 }
 
-const defaultServiceKeySeed = "elephant ears space cowboy octopus rodeo potato cannon pineapple"
+const configPath = "./flow.json"
+
+var (
+	servicePrivateKeyHex     string
+	servicePrivateKeySigAlgo crypto.SignatureAlgorithm
+)
+
+type config struct {
+	Accounts struct {
+		Service struct {
+			Address    string `json:"address"`
+			PrivateKey string `json:"privateKey"`
+			SigAlgo    string `json:"sigAlgorithm"`
+			HashAlgo   string `json:"hashAlgorithm"`
+		}
+	}
+}
+
+func readConfig() config {
+	f, err := os.Open(configPath)
+	Handle(err)
+
+	d := json.NewDecoder(f)
+
+	var conf config
+
+	err = d.Decode(&conf)
+	Handle(err)
+
+	return conf
+}
+
+func init() {
+	conf := readConfig()
```

Comment:
The config path is relative, won't this fail for any program importing this package run from any other directory than `flow-go-sdk/examples`?

Reply:
For example the following program panics:
```go
package main

import (
    "fmt"
    _ "github.com/onflow/flow-go-sdk/examples"
)

func main() {
    fmt.Println("hello")
}
```

Reply:
Yeah, the examples are intended to be used from a directory that contains a `flow.json` file. I'd like for it to be a bit more flexible, but the `examples/README.md` encourages users to run both the emulator and the examples themselves from the `examples` directory.

The `examples` package isn't really meant to be imported by anything other than the examples in this directory. Do you think we should still make it a bit more robust?

Reply:
What about a friendlier error message? e.g.

> These examples must be run from a directory that contains a `flow.json` file.

Reply:
If it's documented in the README, I'm good with merging. The more descriptive is a good update too.

Reply:
Cool. Improved the error message here: https://github.com/onflow/flow-go-sdk/pull/49/commits/194382e3412be1d546752359e489d66cab0bfdd5